.github/workflows/upload-testing-image-to-ecr.yaml

```diff
@@ -23,6 +23,9 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
Same comment here, not needed as we don't have any node steps.

However, it has made me think we need to check the Dockerfile!